### ISSUE TITLE
Template Modal is showing close button

### ISSUE THREAD

**Issue by shukerullah** [tinypass/piano-sdk-for-android#13]
Template Modal is showing close button even `eventData.showCloseButton` is set to `false`.

**Comment by shukerullah** [comment]
Any update on this? I have set `SHOW CLOSE BUTTON` to `NO`, but still I can see the close button.  Even `eventData.showCloseButton` is `false`.

<img width="330" alt="Screen Shot 2020-11-16 at 12 08 11 PM" src="https://user-images.githubusercontent.com/963765/99227291-78681780-2804-11eb-926a-bdf15239524c.png">

**Comment by DeKaN** [comment]
At Android `showCloseButton` changes only fragment's property `isCancelable`. I can't see your template, maybe your template contains a dedicated button. By the way, did you try to write to the [support team](mailto:support@piano.io)?

**Comment by shukerullah** [comment]
Here is my template. I will write this to [support team](support@piano.io) also.

```
<config close-button-type="boilerplate"></config>
<div class="pn-boilerplate [%% bgClass %%]">
  <div class="pn-boilerplate__header">
    <div class="pn-boilerplate__header-controls">
      <language-selector
        class="pn-boilerplate__language-selector"
      ></language-selector>
    </div>
    <button
      ng-if="isShowBoilerplateCloseButton()"
      class="pn-boilerplate__close unbutton"
      type="button"
      ng-click="close()"
      external-event="meter-close-modal"
    >
      <span class="visually-hidden"><t>Close</t></span>
      <svg
        class="pn-boilerplate__close-icon"
        width="12"
        height="12"
        viewBox="0 0 12 12"
        fill="none"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          fill-rule="evenodd"
          clip-rule="evenodd"
          d="M4.586 6L.293 1.707 1.707.293 6 4.586 10.293.293l1.414 1.414L7.414 6l4.293 4.293-1.414 1.414L6 7.414l-4.293 4.293-1.414-1.414L4.586 6z"
          fill="#0F0F15"
        />
      </svg>
    </button>
  </div>
  <!-- pn-boilerplate__body -->
</div>
```

**Comment by DeKaN** [comment]
You place the `<button>` tag in your html, so you see the button.  `showCloseButton` allows/denies canceling dialog at Android and shows/hides an additional button for closing at iOS

**Comment by shukerullah** [comment]
If you see my template code there is condition `ng-if="isShowBoilerplateCloseButton()"`, which basically will handle close button. This template is working fine on `Web`. However, on app `isShowBoilerplateCloseButton` is not triggered.

**Comment by shukerullah** [comment]
Any update on this?

I have a template that dynamically show some buttons and this template is working fine on web but on app it seem like angular part is not working.

> If you see my template code there is condition `ng-if="isShowBoilerplateCloseButton()"`, which basically will handle close button. This template is working fine on `Web`. However, on app `isShowBoilerplateCloseButton` is not triggered.

**Comment by DeKaN** [comment]
Has been fixed in 2.0.6
